internal/pkg/template/templates/workloads/services/backend/manifest.yml

```diff
@@ -57,4 +57,6 @@ exec: true     # Enable running commands in your container.
 #  test:
 #    count: 2               # Number of tasks to run for the "test" environment.
 #    deployment:            # The deployment strategy for the "test" environment.
-#       rolling: 'recreate' # Stops existing tasks before new ones are started for faster deployments.
\ No newline at end of file
+#       rolling: 'recreate' # Stops existing tasks before new ones are started for faster deployments.
+storage:
+  readonly_fs: true
```

Comment:
Sweet, can we also add a comment next to it (aligned like the other fields), saying something like:
```yaml
storage:
   readonly_fs: true       # Limit to read-only access to mounted root filesystems by default.
```

Reply:
Do we need a check to write this value if `{{- if .Platform.PlatformString}}` is not Windows?

Reply:
I will only populate readonly_fs field to true in the manifest file only when the platform is not Windows. If it is not Windows I will not at all populate this manifest field. Does this sounds good?